Frame.cc

```diff
@@ -1012,20 +1018,20 @@ bool Frame::getSublatticeXY(casacore::SubLattice<float>& sublattice, std::pair<i
 bool Frame::getSpectralData(std::vector<float>& data, casacore::SubLattice<float>& sublattice, int checkPerChannels) {
     bool dataOK(false);
     casacore::IPosition sublattShape = sublattice.shape();
+    casacore::IPosition count(sublattShape);
+    int profileAxis = -1; // profile axis index
```

Comment:
(spectral) profileAxis is already determined and saved as local variable spectralAxis, and number of channels (profileSize) is local variable nchan (function nchannels())

Reply:
Done.